style.css

```diff
@@ -0,0 +1,69 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+main{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    min-height: 100vh;
+
+    background: #042014;
+
+    animation: animateBg 10s linear infinite;
+}
+@keyframes animateBg{
+    0%{
+        filter: hue-rotate(0deg);
+    }
+    100%{
+        filter: hue-rotate(360deg);
+    }
+}
+main .loader{
+    position: relative;
+
+    width: 120px;
+    height: 120px;
+} 
+main .loader span{
+    position: absolute;
+    inset: 0;
+
+    width: 100%;
+    height: 100%;
```

Comment:
if you remove this then what will happen ??
you already using an inset so I think this height and width don't make any change you can remove (width and height).

Reply:
ok.